src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_RANDOM_VALUE = 100;
+
+    public String getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        Figure square = new Square(random.nextInt(MAX_RANDOM_VALUE),
+                colorSupplier.getRandomColor());
+        Rectangle rectangle = new Rectangle(random.nextInt(MAX_RANDOM_VALUE),
+                random.nextInt(MAX_RANDOM_VALUE), colorSupplier.getRandomColor());
+        Circle circle = new Circle(random.nextInt(MAX_RANDOM_VALUE),
+                colorSupplier.getRandomColor());
+        Figure[] figures = {square, rectangle, circle};
```

Comment:
please rewrite this using switch case
use all of your figures, not only 3
move things like random and color supplier to field level